examples/dockerdev/.dockerdev/Dockerfile

```diff
@@ -56,7 +56,9 @@ ENV LANG=C.UTF-8 \
 
 # Upgrade RubyGems and install required Bundler version
```

Comment:
Could you please add a link to this PR to the comment for clarification?

Reply:
is it okay now?